Save.bb

```diff
@@ -1507,6 +1507,9 @@ Function LoadGameQuick(file$)
 			NTF_1499PrevRoom = r
 			Exit
 		EndIf
+		If r\RoomTemplate\Name = "gatea" Then
+			PositionEntity r\obj,EntityX(r\obj),r\y,EntityZ(r\obj)
+		EndIf

```

Comment:
This doesn't fix the actual problem of Gate A moving upwards in the first place

Reply:
I think GateA moves upwards so that it doesn't interfere with the map (I didn't porgrammed it like that). I only reset the position after a quickload so that it can launch upwards again (if you enter GateA). Otherwise, it keeps the old position and it would go up everytime you enter GateA (when you load the game again).
I did it like that and it fixed the issue, so I guess this works.

Reply:
The proper solution would to either place Gate A at some other location so moving upwards wouldn't be necessary, or just making it move upwards on map generation instead of whenever the hell it's deciding to do it now.

Reply:
Yeah, that might be a better solution. I consider that and see when I want to change it.

Reply:
The room seems to be moved up at the beginning of the "gatea" event (and I have no idea why, if it intersects with the rest of the map it should already intersect before the player enters it). In any case I agree with juanjp600, the code is already so full of hacky workarounds that only fix the symptoms of a bug instead of the actual issue that we don't need a single one more. Especially in this case when the underlying issue should be fairly straightforward to fix.

Reply:
Yes, I could fix GateA's position altogether.
Altough I myself didn't programmed that Gate A goes up everytime. I made this workaround because I don't know if GateA would still work if it doesn't get launched up or if it actually works.
Only way to find this out is to test it myself.